diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -371,20 +378,27 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
         SECONDARY_CONSTRUCTOR -> parent(CLASS)!!
         else -> treeParent
     }
-        .getIdentifierName()?.text
+            .getIdentifierName()?.text
 
     private fun ASTNode.getOrderedCallExpressions(psi: PsiElement, result: MutableList<ASTNode>) {
         // if statements here have the only right order - don't change it
 
-        if (psi.children.isNotEmpty() && (psi.children[0].node.elementType != DOT_QUALIFIED_EXPRESSION
-                        && psi.children[0].node.elementType != SAFE_ACCESS_EXPRESSION )) {
-            result.add(psi.children[0].node.siblings(true)
+        if (psi.children.isNotEmpty() && (psi.firstChild.node.elementType != DOT_QUALIFIED_EXPRESSION
+                        && psi.firstChild.node.elementType != SAFE_ACCESS_EXPRESSION)) {
+                            val firstChild = psi.firstChild
+                            if (firstChild.node.elementType == POSTFIX_EXPRESSION) {
+                                if (firstChild.firstChild.node.elementType == DOT_QUALIFIED_EXPRESSION
+                                        || firstChild.firstChild.node.elementType == SAFE_ACCESS_EXPRESSION)
```

Comment:
You can extract this comparison to a separate method, it already occurs two times (`psi.firstChild.node.elementType ...`)